docs/examples/physics_using_pybullet/viz_domino.py

```diff
@@ -0,0 +1,238 @@
+# The following imports are necessary for physics simulations
+import numpy as np
+from fury import window, actor, ui, utils
+import itertools
+import pybullet as p
+
+# Instantiate Pybullet client.
+client = p.connect(p.DIRECT)
+
+# Apply gravity to the scene.
+gravity_x = 0
+gravity_y = 0
+gravity_z = -10
+p.setGravity(gravity_x, gravity_y, gravity_z, physicsClientId=client)
+
+
+# Base Plane Parameters
+base_size = np.array([15, 15, 0.2])
+base_color = np.array([1, 1, 1])
+base_position = np.array([0, 0, -0.1])
+base_orientation = np.array([0, 0, 0, 1])
+
+
+# Creating BASE Plane
+base_actor = actor.box(centers=np.array([[0, 0, 0]]),
+                       directions=[0, 0, 0],
+                       scales=base_size,
+                       colors=base_color)
+
+base_coll = p.createCollisionShape(p.GEOM_BOX,
+                                   halfExtents=base_size / 2)
+
+# half of the actual size.
+base = p.createMultiBody(
+    baseCollisionShapeIndex=base_coll,
+    basePosition=base_position,
+    baseOrientation=base_orientation)
+
+p.changeDynamics(base, -1, lateralFriction=0.3, restitution=0.5)
+
+
+# Brick Parameters
+brick_mass = 0.5
+brick_size = np.array([2,0.8, 0.2])
+
+nb_bricks = 10
+brick_centers = np.zeros((nb_bricks, 3))
+
+# Keep all the Bricks Parallel
+brick_directions = np.zeros((nb_bricks, 3))
+brick_directions[:] = np.array([1, 0, 1])
+
+brick_orns = np.zeros((nb_bricks, 4))
+
+brick_sizes = np.zeros((nb_bricks, 3))
+brick_sizes[:] = brick_size
+
+brick_colors = np.random.rand(nb_bricks, 3)
+
+brick_coll = p.createCollisionShape(p.GEOM_BOX,
+                                    halfExtents=brick_size / 2)
+
+# We use this array to store the reference of brick objects in pybullet world.
+bricks = np.zeros(nb_bricks, dtype=np.int8)
+
+
+n_dominoes = 10
+centers_list = np.zeros((n_dominoes, 3))
+
+# Adding the dominoes
+for i in range(n_dominoes):
+    center_pos = np.array([(i*0.99)-2.8,1,0])
+    brick_centers[i] = center_pos
+    brick_orns[i] = np.array([0, 0, 0, 1])
+    bricks[i] = p.createMultiBody(baseMass=brick_mass,
+                                      baseCollisionShapeIndex=brick_coll,
+                                      basePosition=center_pos,
+                                      baseOrientation=brick_orns[i])
+    p.changeDynamics(bricks[i], -1, lateralFriction=0.1, restitution=0.1)
```

Comment:
Your domino position calculation is a bit wrong. Your dominoes are being rendered inside the BASE plate and that why you can observe a weird bump during the start of the simulation.

Reply:
@Nibba2018 Yeah This is the place I am facing problem.. Now I have changed the Position and There is no bump in the start of the Simulation. **But I have a doubt why are the Domino's going below below the Base during Simulation?** I have kept center of the Base as `(0,0,0)`. Since Size of the bricks is `np.array([2,0.8, 0.2])` I have kept the y coordinate of the Bricks as 0.4 now like this -
```
for i in range(n_dominoes):
    center_pos = np.array([(i*0.99)-3.3,0.4,1.5])
```

**So Now when I am running the Code, Initially the bricks are lying exactly with their bottom ends on the base, But when Simulation is proceeding, The bottom ends of Bricks are going below the base !!!**

Please check the file now(I made commit in the same PR) and help me resolve this 

Thanks !

Reply:
![image](https://user-images.githubusercontent.com/29832615/97733577-0c946800-1afe-11eb-9df6-08bf00bdf1c8.png)
@SanjayMarreddi, this is how your dominoes are being initialised in pybullet.

Reply:
I think you are getting your orientation wrong.

Reply:
@Nibba2018 
I have corrected the Orientation Problem. **Now The Simulation is working Correctly in Pybullet World (GUI option) !**
But in the FURY World, The Dominoes are coming in Horizontal position because of which the Simulation in FURY world is not completely Correct. Where as in the Pybullet world they are fine and in Vertical Position.
I have committed my changes in the same PR.
Could you please tell me how to approach in resolving this !!

Thanks !